apps/frontend/src/components/ShowMembers/ShowMembers.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+
+function App() {
+  const usernames = ["radu", "antonina", "ecem", "maria", "mike", "john"];
+
+  return (
+    <div>
+      <UserCircleStack usernames={usernames} />
+    </div>
+  );
+}
+
+function UserCircleStack({ usernames }) {
```

Comment:
The file and folder name should have the same name as the exported component. In this case, your file name and folder name should be 'UserCircleStack'

Reply:
Sorry for doing it so late. I did as you mentioned. I hope it's true. :)